### DISCUSSION TITLE
Remove dependency on TARDISWeepingAngels

### DISCUSSION THREAD

**Discussion by DalekCraft2** [eccentricdevotion/TARDIS#553] [General]
TWA is an addon for TARDIS, but TARDIS hooks into it when it should be the other way around. If some sort of API was made for the things what TWA does in TARDIS, then TWA could use TARDIS (or just TCG, as it does now) as a dependency instead.

This is not really necessary, but it would be convenient.

**Comment by eccentricdevotion** [comment]
> TWA is an addon for TARDIS

TARDISWeepingAngels is a standalone plugin

> TARDIS hooks into it when it should be the other way around

TARDIS depends on some of the features of TWA, but not vice versa, so it's like any other dependency - don't be confused just because it has "TARDIS" in its name
